tools/CoqMakefile.in

```diff
@@ -39,9 +39,6 @@ OCAMLFIND         := $(COQMF_OCAMLFIND)
 CAMLFLAGS         := $(COQMF_CAMLFLAGS)
 HASNATDYNLINK     := $(COQMF_HASNATDYNLINK)
 
-@CONF_FILE@: @PROJECT_FILE@
-	@COQ_MAKEFILE_INVOCATION@
```

Comment:
You could just add `[ - f @CONF_FILE@ ]` (maybe + some nice error message) before the invocation, then if the conf file is present it can be updated but if it's missing you get an error.

Reply:
Or something using makefile's `wildcard` function